examples/create_attribute/create_attribute.py

```diff
@@ -10,7 +10,7 @@
 Full documentation is provided at http://python.dronekit.io/examples/create_attribute.html
 """
 
-from dronekit import connect
+from dronekit import connect, Vehicle
```

Comment:
Do we really need a model where we separately import from dronekit and dronekit.lib? Would be cool to not have to think about where you import from.